tests/typescript_ambient/ambient.ts

```diff
@@ -0,0 +1,15 @@
+declare module "classnames" {
+  export default function classnames(...inputs: (string | number | false | object | undefined)[]): string;
```

Comment:
body is not allowed in an ambient declaration. (i.e. `declare module "..."`). here's a [failing ts build](https://www.typescriptlang.org/play/index.html#src=declare%20module%20%22classnames%22%20%7B%0A%20%20%20%20export%20default%20function%20classnames(%0A%20%20%20%20%20%20%20%20...inputs%3A%20(string%20%7C%20number%20%7C%20false%20%7C%20object%20%7C%20undefined)%5B%5D%0A%20%20%20%20)%3A%20string%20%7B%20return%20%22string%22%3B%20%7D%0A%7D%0A): 
```tsx
declare module "classnames" {
    export default function classnames(
        ...inputs: (string | number | false | object | undefined)[]
    ): string { return "string"; }
}
```
> An implementation cannot be declared in ambient contexts.